### ISSUE TITLE
add-manifest.json missing

### ISSUE THREAD

**Issue by bmann** [fission-suite/web-api#62]
Via Yaz in Discord 

```
Installing library in /Users/yazkhoury/Desktop/Github/ETC/ipfs-api/.stack-work/install/x86_64-osx/bdda0508a06d206cbee8b11de88fb653776e843fa6ba5eb3a1e423936d74a573/8.6.5/lib/x86_64-osx-ghc-8.6.5/ipfs-api-1.0.2-UVn94bUTmFGgkHViQksuM
Installing executable server in /Users/yazkhoury/Desktop/Github/ETC/ipfs-api/.stack-work/install/x86_64-osx/bdda0508a06d206cbee8b11de88fb653776e843fa6ba5eb3a1e423936d74a573/8.6.5/bin
Registering library for ipfs-api-1.0.2..
Completed 186 action(s).
server: ./addon-manifest.json: openBinaryFile: does not exist (No such file or directory)
```

**Comment by expede** [comment]
It is intentionally excluded — it includes shared secrets with Heroku. I'll add an example one to get the app loading, but deploying this to work with Heroku will need someone to actually go through the add on creation process.

**Comment by bmann** [comment]
@expede probably just a docs issue or we include with fill in the blank values. 

Basically “you can ignore this if you aren’t running a Heroku add on”.

**Comment by expede** [comment]
@bmann the app will (intentionally) fail to load without these variables set. We can do a version that doesn't require Heroku, but the shortest path is a dummy config. I _can_ provide a startup flag of `--no-heroku` or something. It's not a ton of work or anything, but feels like more effort in that direction than there needs to be when we have a work around.

**Comment by YazzyYaz** [comment]
I think just having a sample file in order to run would make the most sense. I'm interested in trying to containerize it, so understanding what Heroku is doing and if it's easy to just isolate that bit when I containerize it would help.

**Comment by expede** [comment]
@YazzyYaz Sounds great! I've added an [example manifest](https://github.com/fission-suite/web-api/blob/master/addon-manifest.json.example) to the repo. Just update the `CHANGE ME` fields and remove the `.example` from the file name, and it should be good to go! Please let me know if you run into further troubles 👍

**Comment by expede** [comment]
Closed by #65
